docs/reference/rfc3977-response-codes.md

```
@@ -0,0 +1,123 @@
+
+Complete reference from [RFC 3977 §3.2.1](https://datatracker.ietf.org/doc/html/rfc3977#section-3.2.1) and [RFC 4643 (Authentication)](https://datatracker.ietf.org/doc/html/rfc4643).
+
+## 1xx - Informational (All Multiline)
+
+- **100** - Help text follows
+- **101** - Capabilities list follows
+- **111** - Date (DATE command response)
+
+## 2xx - Success (Selected Multiline)
+
+### Connection/Session
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)